### ISSUE TITLE
Broken in Blender 2.83

### ISSUE THREAD

**Issue by CastelHQ** [johnzero7/XNALaraMesh#63]
```
------------------------------------------------------------
---------------EXECUTING XPS PYTHON IMPORTER----------------
------------------------------------------------------------
Importing file:  D:\Downloads\****.mesh.ascii
rootDir: D:\Downloads\****
Reading Bones
Reading Meshes
Importing Armature 6 bones

---*** Importing Mesh Submesh0_0 ***---
UV Layer Count: 4
Texture Count: 1
Created Mesh: Submesh0_0
New Mesh = Submesh0_0
Traceback (most recent call last):
  File "C:\Users\****\AppData\Roaming\Blender Foundation\Blender\2.83\scripts\addons\XNALaraMesh-master\xps_tools.py", line 164, in execute
    status = import_xnalara_model.getInputFilename(xpsSettings)
  File "C:\Users\****\AppData\Roaming\Blender Foundation\Blender\2.83\scripts\addons\XNALaraMesh-master\import_xnalara_model.py", line 78, in getInputFilename
    status = xpsImport()
  File "C:\Users\****\AppData\Roaming\Blender Foundation\Blender\2.83\scripts\addons\XNALaraMesh-master\import_xnalara_model.py", line 159, in xpsImport
    meshes_obs = importMeshesList(armature_ob)
  File "C:\Users\****\AppData\Roaming\Blender Foundation\Blender\2.83\scripts\addons\XNALaraMesh-master\import_xnalara_model.py", line 478, in importMeshesList
    for meshInfo in newMeshes]
  File "C:\Users\****\AppData\Roaming\Blender Foundation\Blender\2.83\scripts\addons\XNALaraMesh-master\import_xnalara_model.py", line 478, in <listcomp>
    for meshInfo in newMeshes]
  File "C:\Users\****\AppData\Roaming\Blender Foundation\Blender\2.83\scripts\addons\XNALaraMesh-master\import_xnalara_model.py", line 580, in importMesh
    bpy.context.view_layer.objects.active = mesh_ob
RuntimeError: Error: ViewLayer 'View Layer' does not contain object 'Submesh0_0'


location: <unknown location>:-1

location: <unknown location>:-1
```

The same model imports fine into Blender 2.82

**Comment by ldo** [comment]
Copy and paste the text, don’t screenshot it.

**Comment by CastelHQ** [comment]
Done

**Comment by CastelHQ** [comment]
@johnzero7 commenting out line 580 in import_xnalara_model.py gets rid of the error.

**Comment by johnzero7** [comment]
Thanks for the report. It is fixed in release [2.0.2](https://github.com/johnzero7/XNALaraMesh/releases/tag/v2.0.2)
